lib/java/src/crossTest/java/org/apache/thrift/test/TestClient.java

```diff
@@ -823,4 +808,10 @@ public static void main(String[] args) {
 
     System.exit(returnCode);
   }
+
+  private static byte[] getBytesData() {
+    ByteArrayOutputStream byteArrayOutputStream = new ByteArrayOutputStream(256);
+    IntStream.range(-128, 128).forEach(byteArrayOutputStream::write);
+    return byteArrayOutputStream.toByteArray();
+  }
```

Comment:
These seem unrelated, but it's a nice improvement. None of this requires Java 11, right?

Reply:
no, otherwise the cross-test CI in GitHub workflow will fail